### DISCUSSION TITLE
Copy 'persisted-documents.json' after it is built

### DISCUSSION THREAD

**Discussion by onionhammer** [dotansimha/graphql-code-generator#9801] [Q&A]
I've got a `graphql-codegen --watch` running, and it seems plugins run before that file is generated, so I'm wondering if there's a way to tap into the generation of that file and copy it somewhere else

**Comment by SagnikPradhan** [comment]
It’s problematic for monorepo projects that persisted query manifests can only be generated via the client preset and that their location isn’t configurable. I ended up working around this by copying the generated manifest

```diff
	hooks: {
+		afterAllFileWrite: [
+			() => exec("cp ... ...").then(() => {}),
```
